src/Histogram.tsx

```diff
@@ -105,6 +105,7 @@ export interface IHistogramProps {
   duration: number;
   colorScheme: string[];
   domain: IDomain;
+  downloadChartId?: string;
```

Comment:
would this make sense to be changed just to 'id' ? Then we can reference the chart for other things than just downloading?

Reply:
Yep definitely - updating now